wouso/games/teamquest/models.py

```diff
@@ -28,15 +28,55 @@ class TeamQuestGroup(PlayerGroup):
     def is_empty(self):
         return self.users.count() < 1
 
+    def is_full(self):
+        return self.users.count() == 4
```

Comment:
Add a max users attribute in the TeamQuestGroup class and replace this "4" with it.